lib/v2/gov/moe/moe.js

```diff
@@ -45,27 +45,33 @@ module.exports = async (ctx) => {
                     const firstA = item.find('a');
                     const itemUrl = new URL(firstA.attr('href'), moeUrl).href;
 
-                    const infos = await ctx.cache.tryGet(itemUrl, async () => {
-                        const res = {};
-                        const response = await got(itemUrl);
-                        const data = cheerio.load(response.data);
+                    let infos
```

Comment:
```suggestion
                    let infos;
```
Lint your code before pushing pls.